app/models/manageiq/providers/autosde/inventory/collector/target_collection.rb

```diff
@@ -67,7 +67,6 @@ def parse_targets!
       case target
       when PhysicalStorage
         add_target(:physical_storages, target.ems_ref)
-        add_target(:physical_storage_families, PhysicalStorageFamily.find(target.physical_storage_family_id).ems_ref)
```

Comment:
Since this is an existing record I don't know if we don't need to add a target for the physical_storage_family.
This would be required if
1. This were a new physical_storage and it is possible it has a new physical_storage_family that we haven't refreshed yet and thus the lazy find would fail
2. You want to capture changes to the physical_storage_family by refreshing the physical_storage, not sure why you wouldn't just target the physical_storage_family in that case though

Reply:
Agree, this can be removed, we dont need this for now, the target refresh is only set for editing currently